src/Base58.mo

```diff
@@ -13,6 +14,131 @@ module {
     's','t','u','v','w','x','y','z'
   ];
 
+  private let mapBase58 : [Nat8] = [
+    255,255,255,255,255,255,255,255, 255,255,255,255,255,255,255,255,
+    255,255,255,255,255,255,255,255, 255,255,255,255,255,255,255,255,
+    255,255,255,255,255,255,255,255, 255,255,255,255,255,255,255,255, 255, 0,
+    1, 2, 3, 4, 5, 6, 7, 8,255,255,255,255,255,255, 255, 9,10,11,12,13,14,15,
+    16,255,17,18,19,20,21,255, 22,23,24,25,26,27,28,29,
+    30,31,32,255,255,255,255,255, 255,33,34,35,36,37,38,39,
+    40,41,42,43,255,44,45,46, 47,48,49,50,51,52,53,54,
+    55,56,57,255,255,255,255,255, 255,255,255,255,255,255,255,255,
+    255,255,255,255,255,255,255,255, 255,255,255,255,255,255,255,255,
+    255,255,255,255,255,255,255,255, 255,255,255,255,255,255,255,255,
+    255,255,255,255,255,255,255,255, 255,255,255,255,255,255,255,255,
+    255,255,255,255,255,255,255,255, 255,255,255,255,255,255,255,255,
+    255,255,255,255,255,255,255,255, 255,255,255,255,255,255,255,255,
+    255,255,255,255,255,255,255,255, 255,255,255,255,255,255,255,255,
+    255,255,255,255,255,255,255,255, 255,255,255,255,255,255,255,255,
+    255,255,255,255,255,255,255,255,
+  ];
+
+  public func decode(input : Text) : [Nat8] {
```

Comment:
Could you please add some documentation to the functions, in particular the public ones?

Reply:
Added docs.